src/scraping-defaults.js

```diff
@@ -0,0 +1,13 @@
+const SCRAPING_DEFAULT_OPTIONS = {
+    http2: true,
+    https: {
+        rejectUnauthorized: false,
+    },
+    throwHttpErrors: false,
+    ciphers: 'TLS_AES_256_GCM_SHA384',
+    useHeaderGenerator: true,
+};
```

Comment:
Would be nice to have comments explaining the defaults here and maybe even how to get the optimal default ciphers in the future or how to check that those are still good.

Reply:
I guess the ciphers are our internal secret 😉